fixtures/js-stubs/accessRequest.tsx

```diff
@@ -0,0 +1,20 @@
+import {AccessRequest as AccessRequestType} from 'sentry/types';
+
+import {Member} from './member';
+import {Team} from './team';
+
+export function AccessRequest(
+  params: Partial<AccessRequestType> = {}
+): AccessRequestType {
+  return {
+    id: '123',
+    member: Member(),
+    team: Team(),
+    requester: params.requester ?? {
```

Comment:
I think that `params.requester ??` will get overridden by the spread below

Reply:
ah yeah, will change